tobias/utils/motifs.py

```diff
@@ -656,31 +666,32 @@ def get_formation(formation, ncol, nrow, nmotifs):
 #Contains info on one motif formatted for use in moods
 class OneMotif:
 
-	bases = ["A", "C", "G", "T"]
-
-	def __init__(self, motifid=None, name=None, counts=None):
+	id = "" # motif id
+	name = "" # motif name
+	bases = ["A", "C", "G", "T"] # alphabet order must correspont with counts matrix!
+	bg = np.array([0.25,0.25,0.25,0.25]) # background set to equal by default
+	strand = ["+", "-"] # both means unknown

```

Comment:
Unknown means that we assume "+" - a motif cannot have two strands. I would prefer to keep the default at "+", because then you don't have to take care of it later on.

Reply:
I adapted this from meme format as it was easier for parsing at a few places. The problem with this "+ could also mean unknown" approach is that the `OneRegion` objects in `scan_sequence()` would always get '+' whereas now they could also get '.' for unknown.
I don't know if this is further utilized but there might be a future use case.

Reply:
Okay, good point about the scan_sequence(). Then maybe you could set the strand to "." - instead of a list of ["+","-"]? Then you keep the functionality but keep the strand to always be a string

Reply:
So to be on the same page, strand should be defined as a string which is one of "+", "-" or "."? Should this be enforced or is it enough to just assume it is one of the three values?

Reply:
Yes exactly - and yes let's enforce it at creation 👍  Then the default can be ".", but OneMotif can also take "strand" as a parameter, which will then be checked